examples/decoding/plot_haxby_decoder.py

```diff
@@ -0,0 +1,59 @@
+"""
+Decoding with decoder metaestimator: face vs house recognition
+==============================================================
+
+Here is a simple example of decoding with a the decoder metaestimator,
+reproducing the Haxby 2001 study on a face vs house discrimination task.
+"""
+
+#########################################################################
+# Load Haxby dataset
+import numpy as np
+import nibabel
+from nilearn import datasets
+from nilearn.image import new_img_like
+
+# Load the Haxby dataset
+from nilearn.datasets import fetch_haxby
+data_files = fetch_haxby(n_subjects=1)
+
+func_filenames = data_files.func[0]
+labels_filenames = data_files.session_target[0]
+
+from nilearn.image import mean_img
+background_img = mean_img(func_filenames)
+
+# Load Target labels
+import numpy as np
```

Comment:
same here already imported above. May be like in other examples it will be good to import at the usage area than top.